### DISCUSSION TITLE
第4节课作业(3班)

### DISCUSSION THREAD

**Discussion by matrixgame2018** [InternLM/Tutorial#248] [作业]
第4节课作业(3班)

**Comment by 443752009** [comment]
http://t.csdnimg.cn/k3CZ4

**Comment by SchweitzerGAO** [comment]
https://schweitzergao.github.io/2024/01/10/mmtutorial-4-assignment/

**Comment by wm901115nwpu** [comment]
https://zhuanlan.zhihu.com/p/677563702

**Comment by Tallisgo** [comment]
http://t.csdnimg.cn/GYNtF

**Comment by semple030228** [comment]
https://blog.csdn.net/m0_49289284/article/details/135580086

**Comment by GoooHi** [comment]
https://zhuanlan.zhihu.com/p/677741393

**Comment by Gty-2002** [comment]
https://npumgk6rgc.feishu.cn/docx/PXrFdh5aaobSozx4rmscuMVunih?from=from_copylink

**Comment by xiexiaoshinick** [comment]
[[笔记和作业]XTuner 单卡低成本微调实战](https://zhuanlan.zhihu.com/p/677766199)

**Comment by GoldWaterFall** [comment]
https://www.yuque.com/woshijinshui/wwi5m1/un24hi919efxggti?singleDoc# 《XTuner 大模型单卡低成本微调实战作业》

**Comment by Sampson425** [comment]
https://blog.csdn.net/m0_49908454/article/details/135692711

**Comment by hydexi0122** [comment]
https://zhuanlan.zhihu.com/p/678950457

**Comment by G-Mie** [comment]
https://blog.csdn.net/YYYYbhjbhkv_/article/details/135866374
